README.md

```diff
@@ -2,21 +2,22 @@ ABQs Financial Wellness Resource Wizard
 ===============================
 Check out our prototype [here] (http://codeforamerica.github.io/abq-financial-wellness-wizard/)
 
-This is a prototype application created by the City of Albuquerque Code for America Fellows. It helps people understand their own financial wellness.
+This is a prototype application created by the City of Albuquerque Code for America Fellows. It helps people understand their own financial wellness by giving them an evaluation and using that data to generate a report and recommendations as to what things to do to help you grow financially.
 
-##Here are some examples of the questions that are asked:
+===============================
+
+#Deploying this application:
 
-**Q: What is your level of financial stress?**
+In order to deploy this code you will need to install the Jekyll gem through your command line with `gem install jekyll` (you need Ruby installed to do this).
 
-**A:** Not Stressed
+Once you've done that you should simply be able to run the Jekyll command `jekyll serve` or `jekyll serve --watch` and see the site hosted on your localhost:4000 (with `--watch` it will listen for updates).
 
-**Q: What do you worry about more, paying bills regularly or saving money for a rainy day or big purchase?**
+In order to deploy the site to your own github repo you'll first need to set up your `gh-pages` branch and make that your default branch in your GitHub repositories settings. That will make your site appear at http:[username].github.io/[repo name].
```

Comment:
This is unnecessary. We're only interested in getting them up and running with our codebase.